core/templates/css/oppia.css

```diff
@@ -4380,8 +4380,9 @@ oppia-noninteractive-link {
   border-radius: 4px;
   -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
   box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
-  height: auto;
+  max-height: 125px;
```

Comment:
Hm, no let's not do this. This will fix the height of text fields, this will make scrolling a bit inconvenient for users when they have a large input.